src/gpl/src/graphics.cpp

```diff
@@ -214,12 +214,12 @@ void Graphics::cellPlot(bool pause)
   }
 }
 
-gui::Selected Graphics::select(odb::dbTechLayer* layer, const odb::Point& point)
+gui::SelectionSet Graphics::select(odb::dbTechLayer* layer, const odb::Point& point)
```

Comment:
I would say generally that is true, but if you have several overlapping DRC violations (or routes) then it would not be possible to select the ones "below" the first one selected. Changing this to a set allows the selection in LayoutViewer to cycle through the objects below just like it can with instances.

Reply:
That makes sense though this code doesn't actually seem to do that.

Reply:
ahh, I see, yes, I didn't want to mess around too much in there so I just made it "function" the same as before, until I had a change to look more closely at it or Eder could update once this is merged.